charts/kubescape-operator/templates/node-agent/configmap.yaml

```diff
@@ -1,6 +1,12 @@
 {{- $components := fromYaml (include "components" .) }}
 {{- $configurations := fromYaml (include "configurations" .) }}
 {{- if $components.nodeAgent.enabled }}
+{{/* validate that either alertCRD.scopeClustered or alertCRD.scopeNamespaced defined when the capability enabled */}}
```

Comment:
I try to have all the logic concentrated in _helpers.tpl - see if it makes sense to move that there

Reply:
I tried!
The problem is that when I move it there, it looks like the test is not working (I does not fail templating)

Reply:
ok whatever... you know what I think about Helm